templates/data/options.py

```diff
@@ -15,7 +15,10 @@
 
 # The prefix for the bot (str). Define a list of stings for multiple prefixes.
 # ie: `["?", "!", "pls "]`
-prefix = "?"
+prefix = ["?", "? "]
```

Comment:
```suggestion
prefix = ["? ", "?"]
```
You need the version with the space first, otherwise the command for `? help` will be `_help` (underscore is space)